ow_plexil/src/plexil-adapter/PlexilInterface.h

```diff
@@ -44,6 +46,34 @@ class PlexilInterface
   // comprise all the valid lander operation names.
   std::map<std::string, int> m_runningOperations;
 
+  template <class Client, class Goal, class ResultPtr, class FeedbackPtr>
+  void nullaryAction (int id, const std::string& name,
```

Comment:
```suggestion
  void runNullaryAction (int id, const std::string& name,
```

Reply:
Good suggestion, done in both files that use this.